templates/schedule.html

```diff
@@ -56,7 +56,7 @@ <h3>Tuesday, December 1st</h3>
         {{ macros::row_freeform(time="9:00", title="Welcome and Introduction to day’s schedule", color="light") }}
         {{ macros::row_list(time="9:05",
                             session="2.1",
-                            title="Lightning talks: Introducing groups (each 5 mins, slides are optional, max 3 slides)",
```

Comment:
Perhaps leaving "5 mins" as a suggestion so people have an idea of what "lightning" means here.

Reply:
I agree, but this one talk is 10 minutes ...